mypy/semanal.py

```diff
@@ -2356,6 +2357,67 @@ def is_classvar(self, typ: Type) -> bool:
     def fail_invalid_classvar(self, context: Context) -> None:
         self.fail('ClassVar can only be used for assignments in class body', context)
 
+    def process_module_assignment(self, s: AssignmentStmt) -> None:
+        """Check if s assigns a module an alias name; if so, update symbol table."""
+        # TODO support module alias assignment not in global scope
+        if not self.is_module_scope():
```

Comment:
Now that https://github.com/python/mypy/pull/3450 is merged, you can remove this, or make it less restrictive.